dask_cuda/initialize.py

```diff
@@ -94,6 +94,13 @@ def initialize(
     )
     dask.config.set({"distributed.comm.ucx": ucx_config})
 
+    # partd is used by default for disk-based shuffling, but unsupported by
+    # cuDF/Dask-cuDF. Therefore, we set task-based shuffling instead.
+    try:
+        dask.config.get("shuffle")
+    except KeyError:
+        dask.config.set({"shuffle": "tasks"})
```

Comment:
Maybe we could use [`update_defaults`]( https://docs.dask.org/en/latest/configuration.html#downstream-libraries ) for this?

Reply:
Good idea. Done in https://github.com/rapidsai/dask-cuda/pull/709/commits/f0b386d31eb209d959177aef1384d4851e8c607f .